docs/trace/extending-the-sdk/README.md

```diff
@@ -172,16 +172,17 @@ but using the
 method. These are referred to as "legacy Activity" in this repo. These libraries
 already create activities but they do so by using the `Activity` constructor
 directly, rather than using `ActivitySource.StartActivity` method. These
-activities does not by default runs though the samplers, and will have their
+activities does not by default runs though the sampler, and will have their
```

Comment:
nit: through* 
Could we rephrase it to this: "These activities by default do not run through samplers"?